tests/tests.cpp

```diff
@@ -20,10 +20,33 @@ TEST(tests, issueBar){
 	bool returnValue = func_1;
 	EXPECT_EQ(true, returnValue);
 }
+
+
 extern "C" int craft_main(int argc, char* argv[]);
 
+extern "C" float setJump(int megaJump, float dy);
+TEST(tests, issue_6){
```

Comment:
It would be neater since this pull request is for Issue #32, the extra code from Issue #6 should be just in the pull request for Issue #6 & not here.